### DISCUSSION TITLE
Need support of the use case of recording on multiple machines

### DISCUSSION THREAD

**Discussion by yubrshen** [matthiasn/lotti#963] [Q&A]
I'd like to keep track of my life on different machines at home, and at work. I also want the records on the different machines can be combined into one integrated view.

**Comment by yubrshen** [comment]
I see that in the Preference panel, there is a synchronization sub-panel, I guess that it might be planned for synchronizations of logs?

**Comment by matthiasn** [comment]
Hi Yu, yes that works, I've done the same for a while, syncing from the work machine to my home laptop. The sync panel so far is only for the sync with the mobile app, I will have to simplify a few things and add some documentation. I will prioritize that now. Thanks for requesting this!

**Comment by rschulman** [comment]
Just discovered `meins` and I'm loving it. Sync between work and home (or a fully fledged mobile experience) would be the icing on the cake!

**Comment by matthiasn** [comment]
@rschulman thanks, I will put together some documentation for syncing between two computers. Would you like to test the mobile app? If you use iOS, I can add you to Testflight. For Android though, you'd have to build it yourself.

**Comment by rschulman** [comment]
I'd love to use the mobile app. I'm on Android, but I have the SDK on my home machine so I can give a go at building it.

**Comment by matthiasn** [comment]
@yubrshen @rschulman the Clojure codebase is no longer under active development and I'm working on a rewrite using Flutter, with the exact same codebase and user experience on desktop and mobile, and sync between devices. I'm moving this to a discussion and will add instructions on how to set that up.
